plugins/channelrx/demoddatv/leansdr/dvbs2.h

```diff
@@ -733,7 +736,7 @@ struct s2_frame_receiver : runnable
         // Read PLSCODE
 
         uint64_t plscode = 0;
-        complex<float> pls_symbols[plscodes.LENGTH];
+        complex<float> pls_symbols[PLSCODES_LENGTH];
```

Comment:
Alternatively you could code this:
```c++
complex<float> pls_symbols[s2_plscodes<T>::LENGTH];
```
and this avoids having to define `PLSCODES_LENGTH`. I tried it using clang and it fixes the variable length array issue.

Reply:
oh, right!